android/build.gradle.kts

```diff
@@ -1,132 +1,107 @@
-    natives("com.badlogicgames.gdx:gdx-freetype-platform:${rootProject.extra["gdxVersion"]}:natives-arm64-v8a")
-    natives("com.badlogicgames.gdx:gdx-freetype-platform:${rootProject.extra["gdxVersion"]}:natives-x86")
-    natives("com.badlogicgames.gdx:gdx-freetype-platform:${rootProject.extra["gdxVersion"]}:natives-x86_64")
-    api("com.badlogicgames.box2dlights:box2dlights:${rootProject.extra["box2DLightsVersion"]}")
-    api("com.badlogicgames.ashley:ashley:${rootProject.extra["ashleyVersion"]}")
-    api("com.badlogicgames.gdx:gdx-ai:${rootProject.extra["aiVersion"]}")
-    api("org.jetbrains.kotlin:kotlin-stdlib:${rootProject.extra["kotlinVersion"]}")
+    implementation(project("core"))
+    api("com.badlogicgames.gdx:gdx-backend-android:${Versions.gdx}")
+    natives("com.badlogicgames.gdx:gdx-platform:${Versions.gdx}:natives-armeabi")
+    natives("com.badlogicgames.gdx:gdx-platform:${Versions.gdx}:natives-armeabi-v7a")
+    natives("com.badlogicgames.gdx:gdx-platform:${Versions.gdx}:natives-arm64-v8a")
+    natives("com.badlogicgames.gdx:gdx-platform:${Versions.gdx}:natives-x86")
+    natives("com.badlogicgames.gdx:gdx-platform:${Versions.gdx}:natives-x86_64")
+    api("com.badlogicgames.gdx:gdx-box2d:${Versions.gdx}")
+    natives("com.badlogicgames.gdx:gdx-box2d-platform:${Versions.gdx}:natives-armeabi")
+    natives("com.badlogicgames.gdx:gdx-box2d-platform:${Versions.gdx}:natives-armeabi-v7a")
+    natives("com.badlogicgames.gdx:gdx-box2d-platform:${Versions.gdx}:natives-arm64-v8a")
+    natives("com.badlogicgames.gdx:gdx-box2d-platform:${Versions.gdx}:natives-x86")
+    natives("com.badlogicgames.gdx:gdx-box2d-platform:${Versions.gdx}:natives-x86_64")
+    api("com.badlogicgames.gdx:gdx-freetype:${Versions.gdx}")
+    natives("com.badlogicgames.gdx:gdx-freetype-platform:${Versions.gdx}:natives-armeabi")
+    natives("com.badlogicgames.gdx:gdx-freetype-platform:${Versions.gdx}:natives-armeabi-v7a")
+    natives("com.badlogicgames.gdx:gdx-freetype-platform:${Versions.gdx}:natives-arm64-v8a")
+    natives("com.badlogicgames.gdx:gdx-freetype-platform:${Versions.gdx}:natives-x86")
+    natives("com.badlogicgames.gdx:gdx-freetype-platform:${Versions.gdx}:natives-x86_64")
+    api("com.badlogicgames.box2dlights:box2dlights:${Versions.box2DLight}")
+    api("com.badlogicgames.ashley:ashley:${Versions.ashley}")
+    api("com.badlogicgames.gdx:gdx-ai:${Versions.gdxAI}")
+    //api(kotlin("stdlib"))
 }
 
 android {
-    buildToolsVersion("29.0.2")
     compileSdkVersion(29)
+
     sourceSets {
         getByName("main").apply {
-            manifest.srcFile("AndroidManifest.xml")
-            java.srcDirs("src")
-            aidl.srcDirs("src")
-            renderscript.srcDirs("src")
-            res.srcDirs("res")
-            assets.srcDirs("assets")
-            jniLibs.srcDirs("libs")
+            java.srcDir("src/main/kotlin")
+            assets.srcDir("../assets")
+            jniLibs.srcDir("libs")
```

Comment:
What happens if `aidl.srcDirs`, `renderscript.srcsDirs` and `res.srcDirs` are not linked to `src/main/kotlin`? Is this an issue?

Seems like res files e.g. got moved to `src/main/kotlin/res` - is that ok?

Reply:
Res should be in `src/main/res` but isn't src the default src folder?